src/it/java/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/EmptyCatchBlockTest.java

```diff
@@ -78,10 +78,10 @@ public void testViolationsByComment() throws Exception {
     @Test
     public void testViolationsByVariableName() throws Exception {
         final String[] expected = {
-            "19: " + getCheckMessage(EmptyCatchBlockCheck.class, "catch.block.empty"),
-            "35: " + getCheckMessage(EmptyCatchBlockCheck.class, "catch.block.empty"),
-            "51: " + getCheckMessage(EmptyCatchBlockCheck.class, "catch.block.empty"),
-            "58: " + getCheckMessage(EmptyCatchBlockCheck.class, "catch.block.empty"),
+            "19:11: " + getCheckMessage(EmptyCatchBlockCheck.class, "catch.block.empty"),
+            "35:11: " + getCheckMessage(EmptyCatchBlockCheck.class, "catch.block.empty"),
+            "51:11: " + getCheckMessage(EmptyCatchBlockCheck.class, "catch.block.empty"),
+            "58:11: " + getCheckMessage(EmptyCatchBlockCheck.class, "catch.block.empty"),
```

Comment:
violation message - `Empty catch block.`([link](https://checkstyle.org/config_blocks.html#EmptyCatchBlock_Violation_Messages))
position 11 is for `catch` token
Problem is code is actually should be placed at `{}`.
Please do update in code to place first SLIST token from catch.

Such more precise placing will be used in auto-fix functionality to with suggestion on fix.

Reply:
Issue is block of catch. A catch should always have a block. If we violate on the catch or the block, it all leads to the same area. I would question why auto-fix, when done, couldn't make the same determination.

I don't mind this change and am fine with it.

@romani One area of thought is how this affects xpath. SLIST is a child of LITERAL_CATCH, but this is a slightly breaking change for current xpaths. Some xpaths examine parent/child and this would make those current suppressions not working.
````
    |      |  `--LITERAL_CATCH -> catch [3:7]
    |      |      |--LPAREN -> ( [3:13]
    |      |      |--PARAMETER_DEF -> PARAMETER_DEF [3:14]
    |      |      |  |--MODIFIERS -> MODIFIERS [3:14]
    |      |      |  |--TYPE -> TYPE [3:14]
    |      |      |  |  `--IDENT -> Exception [3:14]
    |      |      |  `--IDENT -> e [3:24]
    |      |      |--RPAREN -> ) [3:25]
    |      |      `--SLIST -> { [3:27]
    |      |          `--RCURLY -> } [3:28]
````

Reply:
This Check does not support xpath now, it will be supported after this fix.  I am not sure what breaking changes you mention.

Reply:
It can be ignored. I forgot what this change is doing for us. :)

Reply:
@gaurabdg , please update code to put violation on braces.

Reply:
@romani In some of the input files for `EmptyCatchBlockTest.java` have `//warn` comments on the catch token line and not the block/lcurly line. It seems like it was designed to have the violation against the catch token everywhere. Do you want me to change all of them?

Reply:
@romani done, please review.